### ISSUE TITLE
How to run this thing?

### ISSUE THREAD

**Issue by harshil2105** [marcinguy/android712-blueborne#1]
Is this the complete work?

Fully functional payload?

And most important.....how to run this payload provided by you?

**Comment by marcinguy** [comment]
Yes, it is complete, it is mostly Armis code. My work was to find the right offsets for my mobile (Samsung S3 Neo on LinageOS 14.1). You need to adjust certain offsets for your target. There is also variable for payload, which you can change. If you understand exploitation process, know ASM, Python and GDB you should be able to adjust the values for your mobile.
